feat(content): Culture Conversation for native life on Glaze (#11101)
endless-sky/endless-sky: data/human/culture conversations.txt

			label "sestor beaten at farpoint"
			`	Web comments and news anchors seem to be particularly excited to comment on the recent victory of the Navy over the mysterious armada of robotic warships that invaded the Far North. Though at a great cost to the northern fleet garrisons, the invading fleets were successfully stopped at the Farpoint base, preventing the ships from causing any further harm. Veterans of the Farpoint battle seem to be hoarding the spotlight, giving emotional reports of their memory of the fight, and of the sacrifices they went through to achieve victory.`
				goto end

> mission "Exotic Animals - Glaze"

Author: That is a better title for it, thanks
```suggestion
mission "Exotic Life: Glaze"
```

	minor
	source Glaze
	to offer

> 		random < 10

Author: Good call

	on offer
		conversation

> 			`You step to the edge of the mesa from the spaceport, the wind blowing as you take in the view. The canyons stretch out before you, a maze of red and gold stone, their depths lost in shadow. Far in the distance, something moves -- small figures, barely more than specks, making their way through one of the ravines. You squint, but the details remain elusive.`

Author: One day unicode will come (maybe)

```suggestion
			`You step to the edge of the mesa from the spaceport, the wind blowing as you take in the view. The canyons stretch out before you, a maze of red and gold stone, their depths lost in shadow. Far in the distance, something moves - small figures, barely more than specks, making their way through one of the ravines. You squint, but the details remain elusive.`
```

			`	Nearby, a weathered set of credit-operated binoculars stands bolted to the ground, a faded sign above them reading: 5 credits per use.`
			choice
				`	(Take a closer look for 5 credits.)`
					to display
						credits >= 5
				`	(Ignore the specks and return to your ship.)`
					to display
						credits >= 5
					goto closing
				`	(Pat your empty pockets. Return to your ship as you can't even afford the 5 credits.)`
					to display
						credits < 5
					goto closing
			action
				payment -5
			branch norareapes

> 				random > 20

Author: Whoops, that should be flipped. Great idea to set a condition, I'll add that.

				goto end

mission "Exotic Life: Glaze"
	repeat
	minor
	source Glaze
	to offer
		random < 10
		or
			not "chosen sides"
			has "main plot completed"
		not "exotic life: seen glaze apes"
	on offer
		conversation
			`You step to the edge of the mesa from the spaceport, the wind blowing as you take in the view. The canyons stretch out before you, a maze of red and gold stone, their depths lost in shadow. Far in the distance, something moves - small figures, barely more than specks, making their way through one of the ravines. You squint, but the details remain elusive.`
			`	Nearby, a weathered set of credit-operated binoculars stands bolted to the ground, a faded sign above them reading: 1 credit per use.`
			choice
				`	(Take a closer look for 1 credit.)`
					to display
						credits >= 1

> 				`	(Ignore the specks and return to your ship.)`

Reviewer: ```suggestion
				`	(Ignore the specks and return to my ship.)`
```

					to display
						credits >= 1
					goto closing

> 				`	(Pat your empty pockets. Return to your ship as you can't even afford the 1 credit fee.)`

Reviewer: ```suggestion
				`	(Pat my empty pockets and return to my ship as I can't even afford the 1 credit fee.)`
```

					to display
						credits < 1
					goto closing
			action
				payment -1
			branch norareapes
				random < 20
			`	The binoculars hum as you focus in on the figures, expecting hikers. But what you see is something else entirely. A small group of shaggy, long-limbed creatures moves carefully through the canyon. Their fur is the same hue as the sandstone, blending almost perfectly with the terrain. They walk on two legs, but with a hunched, cautious gait, pausing often to glance around before continuing.`

> 			`	One of them stops near a boulder, kneeling to scoop up a handful of dust and let it run through its fingers, as if testing the wind. Another lingers behind the rest, turning its head slightly, as if listening for something. There are no signs of tools or fire, certainly nothing to suggest the "canyon people" of local legend -- but there's an undeniable sense of awareness in the way they move and seem to belong to the landscape itself.`

Reviewer: Getting rid of a duplicate 'as if'.
```suggestion
			`	One of them stops near a boulder, kneeling to scoop up a handful of dust and let it run through its fingers, as if testing the wind. Another lingers behind the rest, turning its head slightly, perhaps listening for something. There are no signs of tools or fire, certainly nothing to suggest the "canyon people" of local legend - but there's an undeniable sense of awareness in the way they move and seem to belong to the landscape itself.`
```

			label "sestor beaten at farpoint"
			`	Web comments and news anchors seem to be particularly excited to comment on the recent victory of the Navy over the mysterious armada of robotic warships that invaded the Far North. Though at a great cost to the northern fleet garrisons, the invading fleets were successfully stopped at the Farpoint base, preventing the ships from causing any further harm. Veterans of the Farpoint battle seem to be hoarding the spotlight, giving emotional reports of their memory of the fight, and of the sacrifices they went through to achieve victory.`
				goto end

mission "Exotic Life: Glaze"
	repeat
	minor
	source Glaze
	to offer
		random < 10
		or
			not "chosen sides"
			has "main plot completed"

> 		and
> 			not "exotic life: seen glaze apes"

Reviewer 2: Everything is implicitly `and` already.
```suggestion
		not "exotic life: seen glaze apes"
```

	on offer
		conversation
			`You step to the edge of the mesa from the spaceport, the wind blowing as you take in the view. The canyons stretch out before you, a maze of red and gold stone, their depths lost in shadow. Far in the distance, something moves - small figures, barely more than specks, making their way through one of the ravines. You squint, but the details remain elusive.`
			`	Nearby, a weathered set of credit-operated binoculars stands bolted to the ground, a faded sign above them reading: 1 credit per use.`
			choice

> 				`	(Take a closer look for 1 credits.)`

Reviewer 3: ```suggestion
				`	(Take a closer look for 1 credit.)`
```

					to display
						credits >= 1
				`	(Ignore the specks and return to your ship.)`
					to display
						credits >= 1
					goto closing
				`	(Pat your empty pockets. Return to your ship as you can't even afford the 1 credit fee.)`
					to display
						credits < 1
					goto closing
			action
				payment -1
			branch norareapes
				random < 20
			`	The binoculars hum as you focus in on the figures, expecting hikers. But what you see is something else entirely. A small group of shaggy, long-limbed creatures moves carefully through the canyon. Their fur is the same hue as the sandstone, blending almost perfectly with the terrain. They walk on two legs, but with a hunched, cautious gait, pausing often to glance around before continuing.`

> 			`	One of them stops near a boulder, kneeling to scoop up a handful of dust and let it run through its fingers, as if testing the wind. Another lingers behind the rest, turning its head slightly, as if listening for something. There are no signs of tools or fire, certainly nothing to suggest the 'canyon people' of local legend -- but there's an undeniable sense of awareness in the way they move and seem to belong to the landscape itself.`

Reviewer 2: Style
```suggestion
			`	One of them stops near a boulder, kneeling to scoop up a handful of dust and let it run through its fingers, as if testing the wind. Another lingers behind the rest, turning its head slightly, as if listening for something. There are no signs of tools or fire, certainly nothing to suggest the "canyon people" of local legend -- but there's an undeniable sense of awareness in the way they move and seem to belong to the landscape itself.`
```

			action

> 					set "exotic life: seen glaze apes"

Reviewer 3: ```suggestion
				set "exotic life: seen glaze apes"
```
